test/e2e/opensearch/00-assert.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 600
+---
+apiVersion: exoscale.crossplane.io/v1
+kind: OpenSearch
+metadata:
+  name: e2e-test-opensearch
+status:
+  atProvider:
```

Comment:
maybe add as well `forProvider:` and some specific opensearch settings to make the tests more complete, you can have a look to other DB tests.

Reply:
can You please explain me why? I can't see added value

Reply:
resolved